Extensions/TweenBehavior/JsExtension.js

```diff
@@ -37,6 +37,65 @@ module.exports = {
       )
       .setExtensionHelpPath('/behaviors/tween');
 
+      const easingChoices = JSON.stringify([
+        'linear',
+        'easeInQuad',
+        'easeOutQuad',
+        'easeInOutQuad',
+        'easeInCubic',
+        'easeOutCubic',
+        'easeInOutCubic',
+        'easeInQuart',
+        'easeOutQuart',
+        'easeInOutQuart',
+        'easeInQuint',
+        'easeOutQuint',
+        'easeInOutQuint',
+        'easeInSine',
+        'easeOutSine',
+        'easeInOutSine',
+        'easeInExpo',
+        'easeOutExpo',
+        'easeInOutExpo',
+        'easeInCirc',
+        'easeOutCirc',
+        'easeInOutCirc',
+        'easeOutBounce',
+        'easeInBack',
+        'easeOutBack',
+        'easeInOutBack',
+        'elastic',
+        'swingFromTo',
+        'swingFrom',
+        'swingTo',
+        'bounce',
+        'bouncePast',
+        'easeFromTo',
+        'easeFrom',
+        'easeTo',
+      ]);
+  
+      extension
+      .addExpression(
+        'Ease',
+        _('Ease'),
+        _('Tween between 2 values according to an easing function.'),
+        '',
+        'res/actions/camera.png'
+      )
+      .addParameter('expression', _('From value'))
+      .addParameter('expression', _('To value'))
+      .addParameter('stringWithSelector', _('Easing'), easingChoices, false)
+      .setDefaultValue('linear')
+      .addParameter('expression', _('Weighting'))
+      .setParameterLongDescription(
+        _('From 0 to 1.')
+      )
+      .getCodeExtraInformation()
+      .setIncludeFile('Extensions/TweenBehavior/shifty.js')
+      .addIncludeFile('Extensions/TweenBehavior/tweenextension.js')
```

Comment:
Nitpicking: let's name this file "tweentools.js"

We use the convention "tools" for functions that are used by events not related to a behavior/object.
This is because these are glue/helpers/tools, but not an extension. The extension itself is in JsExtension.js, but does not exist anymore at runtime (at runtime, the extension concept is gone, it's just a bunch of helper functions).